declarative_authorization.gemspec

```diff
@@ -19,5 +19,5 @@ Gem::Specification.new do |spec|
   spec.metadata['allowed_push_host'] = 'https://rubygems.org'
 
   spec.add_dependency('blockenspiel', ['>= 0.5', '< 1'])
-  spec.add_dependency('rails', ['>= 7', '< 8.1'])
+  spec.add_dependency('rails', ['>= 7.2', '< 8.2'])
```

Comment:
It looks like property is still on rails `7.1.5.1` - is the plan that we wouldn't bump `ae_declarative_authorization` in property until rails is bumped to a newer version that's supported by this gem?

Reply:
oh awesome, thanks for pointing me to that!